assets/content/cookbook/Data structures/ring-array.md

```diff
@@ -113,7 +109,7 @@ It's easy to implement `undo/redo` operations.
 
 class Main {
   static function main() {
-    var h = new History<Int>(3);
+    var h = new History<Int>(4);
     h.add(1);
     h.add(2);
     h.add(3);
```

Comment:
Can't comment on the line below (L116) – `overrided` is not a word, maybe it's best to say `overrides the 1`.